### ISSUE TITLE
loadString() for big .txt files

### ISSUE THREAD

**Issue by Bomel** [processing/p5.js#2661]
#### Most appropriate sub-area of p5.js?

- [ ] Color
- [ ] Core
- [x] Data
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Other (specify if possible)

#### Which platform were you using when you encountered this?

- [ ] Mobile/Tablet (touch devices)
- [x] Desktop/Laptop
- [ ] Others (specify if possible)

This is my first Issue, so plase keep that in mind while reading it.
I tried to load in the `preload()` function a `.txt` file using the function loadString(). The Problem I get is, that in the browser it keeps saing `Loading...`.
The `output_coord_cartesian.txt` is a large file and not loadable in the standart text editors. I think that loading the file just takes very very long in p5.js. If it is so, is there a way around this problem?

Here is my code... maybe it is a diffrent kind of problem 
`sketch.js`:
``` js
var x;
var y;
var t = 0;
var file;


function setup() {
  createCanvas(400, 400);
}

function preload() {
  file = loadStrings("../output_coord_cartesian.txt");
  // also tried moving the file and then
  // file = loadStrings("output_coord_cartesian.txt");
}


function draw() {
  // some magic
}
```
and `index.html`:
``` html
<!DOCTYPE html>
<html>
  <head>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.6.0/p5.js"></script>
    <script src="./sketch.js"></script>
  </head>
  <body>
  </body>
</html>
```

Edit:
Smaller files work fine.

**Comment by Spongman** [comment]
Did you try a smaller file?

**Comment by Bomel** [comment]
Yes I just did. It seems to work fine.

**Comment by limzykenneth** [comment]
@Bomel Can you open the developer console and see if there's any error in there (Cmd + Shift + J in most browsers)

**Comment by Bomel** [comment]
@limzykenneth there seems to be an error message:

The first one is probably not the problem, it is fixed by adding 
```html
<meta charset="UTF-8">
``` 
in `<head>`

Die Zeichenkodierung des HTML-Dokuments wurde nicht deklariert. Das Dokument wird in manchen Browser-Konfigurationen mit verstümmeltem Text dargestellt, wenn das Dokument Zeichen außerhalb des US-ASCII-Bereichs enthält. Die Zeichenkodierung der Seite muss im Dokument oder Transferprotokoll deklariert werden.
index.html



But the second looks more interesting:

TypeError: NetworkError when attempting to fetch resource.
[Weitere Informationen]
p5.js:59833:5

**Comment by limzykenneth** [comment]
How big is the file you are loading? I'll see if I can try to duplicate the issue on my side.

**Comment by Bomel** [comment]
@limzykenneth
Here is the file. I hope you can read/download it. It's size is 706.8 MB
https://drive.google.com/file/d/1TUPcC5HRDnDEF3KM3BdM1hlJvafViRx_/view?usp=drivesdk

**Comment by limzykenneth** [comment]
I finally got to test the file and it works as expected on Safari and Chrome but hit a memory allocation limit on Firefox. I'm not able to replicate your error but I would guess it has something to do with the max file size servable. Depending on your situation, you may need to explore other ways of using data from that file, either off load the lookup to a server or serving smaller chucks of files.

**Comment by Bomel** [comment]
@limzykenneth Thank you, i'll tried to load the file in Firefox. I will try to load it with other Browsers and give you an update :)

**Comment by TanviKumar** [comment]
#3127 solves this

**Comment by lmccart** [comment]
fixed with #3127
